azurerm/resource_arm_action_group.go

```diff
@@ -0,0 +1,320 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/monitor/mgmt/2018-03-01/insights"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmActionGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmActionGroupCreateOrUpdate,
+		Read:   resourceArmActionGroupRead,
+		Update: resourceArmActionGroupCreateOrUpdate,
+		Delete: resourceArmActionGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
Could we add some validation here, like no empty values?

Reply:
Done.